main.js

```diff
@@ -10,4 +10,193 @@ function closeMenu() {
+    popupPara.classList.add('popup_para');
+
+    const popupParaText = document.createTextNode(project.description);
+    popupPara.appendChild(popupParaText);
+
+    popupGridContainer.appendChild(popupPara);
+
+    // popup Buttons part starts here !!!
+
+    const popupButtonMainContainer = document.createElement('div');
+    popupButtonMainContainer.classList.add('popup_button_main_container');
+
+    const popupButtonContainerOne = document.createElement('div');
+    popupButtonContainerOne.classList.add('popup_buttons_container');
+    popupGridContainer.appendChild(popupButtonContainerOne);
+
+    const popupButtonOne = document.createElement('button');
+    popupButtonOne.classList.add('popup_buttons');
+
+    popupButtonOne.addEventListener('click', () => {
+      window.open('https://bilalrajput09.github.io/My-Portfolio/', '_blank');
+    });
+
+    const linkToLive = document.createTextNode(project.linkToLive);
+
+    popupButtonOne.appendChild(linkToLive);
+
+    const popupIconOne = document.createElement('span');
+    popupIconOne.classList.add('popup_icons');
+
+    const popupIconsImgOne = document.createElement('img');
+    popupIconsImgOne.src = './img/see-live.png';
+
+    popupIconOne.appendChild(popupIconsImgOne);
+
+    popupButtonContainerOne.appendChild(popupButtonOne);
+    popupButtonContainerOne.appendChild(popupIconOne);
+
+    const popupButtonContainerTwo = document.createElement('div');
+    popupButtonContainerTwo.classList.add('popup_buttons_container');
+    popupGridContainer.appendChild(popupButtonContainerTwo);
+
+    const popupButtonTwo = document.createElement('button');
+    popupButtonTwo.classList.add('popup_buttons');
+
+    popupButtonTwo.addEventListener('click', () => {
+      window.open('https://github.com/bilalrajput09', '_blank');
+    });
+
+    const linkToSource = document.createTextNode(project.linkToSource);
+
+    popupButtonTwo.appendChild(linkToSource);
+
+    const popupIconTwo = document.createElement('span');
+    popupIconTwo.classList.add('popup_icons');
+
+    const popupIconsImgTwo = document.createElement('img');
+    popupIconsImgTwo.src = './img/github-white-popup.png';
+
+    popupIconTwo.appendChild(popupIconsImgTwo);
+
+    popupButtonContainerTwo.appendChild(popupButtonTwo);
+    popupButtonContainerTwo.appendChild(popupIconTwo);
+
+    popupButtonMainContainer.appendChild(popupButtonContainerOne);
+    popupButtonMainContainer.appendChild(popupButtonContainerTwo);
+
+    popupGridContainer.appendChild(popupButtonMainContainer);
+
+    mainContainer.appendChild(popupGridContainer);
+
+    popupBackgroundContainer.appendChild(mainContainer);
+  });
+  popupHeadingContainer.appendChild(crossButton);
+  body.appendChild(popupBackgroundContainer);
+  crossButton.addEventListener('click', () => {
+    body.removeChild(popupBackgroundContainer);
+  });
+}
+
+const project1 = document.querySelector('.project1');
+project1.addEventListener('click', generatePopup);
```

Comment:
- [ ] Kindly make sure to iterate through the projects array and make sure to display each project's detail every time the `See Project` button is clicked. At this moment, you are only displaying only one project while others are raising errors when we click on the `See Project` button.

- [ ] Kindly make sure to add the event listener to each `See Project` of each project and kindly make sure each pop-card will have unique details different from others.

  - ### APP BEHAVIOUR
   ![chrome-capture-2023-2-8 (2)](https://user-images.githubusercontent.com/87197412/223676436-09cc5498-7214-44c2-a591-ac107297908a.gif)